### PR TITLE
Docker Deployment documentation update

### PR DESCRIPTION
**Pull request by ascholerChemeketa**
Update/streamlining of the docker deployment process documentation

### CODE REVIEW

**Comment by ascholerChemeketa** [comment]
You can preview what it looks like on ReadTheDocs here:
https://runestoneserverascholer.readthedocs.io/en/latest/docker/README.html

File: docker/README.rst [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+    Some of the default books already have "default" courses with the same name as the book. If you try to create
+    a course with a name like ``thinkcspy`` you will be told that the course name is the same as the book.
+
+7. Add a User
+**************************
+
+To add an initial instructor account to the course you have created, you can either create a new user or add
+an existing user as an instructor to the course.
+
+To add a new user, use the ``dmange`` script to run **inituser**. It asks for what class to add the user to and whether or not
```
Comment (bnmnetp) [inline docker/README.rst:277]: typo: manage

File: docker/README.rst [hunk 2]
```diff
@@ -0,0 +1,532 @@
...
+they should be made an instructor.
+
+.. code-block:: bash
+
+    scripts/dmanage inituser
+
+
+Or, if you already have an account that you want to add as an instructor to the new course, you can use the
+``dmange`` script to execute **addinstructor** which will prompt you for a username and course name:
```
Comment (bnmnetp) [inline docker/README.rst:286]: typo: dmanage

File: docker/.env.prototype [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Must be set to hostname of server if you want to access the server from another machine
```
Comment (bnmnetp) [inline docker/.env.prototype:1]: I like this... good addition

**Review by bnmnetp — APPROVED** [review]

**Comment by ascholerChemeketa** [comment]
All issues on my radar fixed.

**Pull request merged**